src/content/docs/configuration/api-keys.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: API Keys
+description: Learn how to create and manage Daytona API keys.
+---
+
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**
+</Label>
+
+Daytona API keys are used to authenticate requests to the [Daytona API](/docs/tools/api). Daytona provides an option to generate an API key for managing your API requests, list all generated API keys, and revoke an API key.
+
+## Generate an API Key
+
+Daytona provides an option to generate API keys.
+
+1. Run the following command to generate an API key:
+
+```bash
+daytona api-key generate
+```
+
+Upon running this command, Daytona will prompt you to enter a name for the API key.
+
+```text
+Name
+>
+```
+
+Upon successful generation of the API key, Daytona will display the following confirmation message:
+
+```text
+You have successfully generated a new API Key and Server URL, which you can find below:
+
+DAYTONA_API_KEY=your-api-key
+DAYTONA_SERVER_URL=your-server-url
+```
+
+Daytona provides an option to connect to the Daytona Server instantly from a client machine by running the following command:
+
+```bash
+daytona profile add -a <your-server-url> -k <your-api-key>
+```
+
+## List API Keys
+
+Daytona provides an option to list all generated API keys.
+
+1. Run the following command to list all generated API keys:
+
+```bash
+daytona api-key list
+```
+
+Upon running this command, Daytona will display the list of all generated API keys.
+
+```text
+Name        API Key
```

Comment:
This is not the correct table that is displayed
![Screenshot 2025-01-13 at 09 51 13](https://github.com/user-attachments/assets/d59fc2c9-2801-46e1-8098-8bcd88a4518f)